test/org/zaproxy/zap/extension/pscanrules/TestInfoPrivateAddressDisclosureUnitTest.java

```diff
@@ -90,15 +90,27 @@ public void alertsIfPrivateIp() throws HttpMalformedHeaderException {
 	}
 
 	@Test
-	public void shouldIgnoreRequestedPrivateIpByDefault() throws HttpMalformedHeaderException {
+	public void shouldIgnoreRequestedPrivateIpByDefault() throws HttpMalformedHeaderException, URIException, NullPointerException {
 		// Given
-		String privateIp = "192.168.36.127";
-		String requestUri = "https://" + privateIp + ":8123/";
-		HttpMessage msg = createHttpMessage(requestUri, privateIp);
-		// When
-		rule.scanHttpResponseReceive(msg, -1, createSource(msg));
-		// Then
-		assertThat(alertsRaised, is(empty()));
+		// ip and aws-hostname which get concatenated with the ports as candidates
+		String[] ipHost = new String[] {"10.0.2.2", "iP-10-0-2-2"};
```

Comment:
The regular expression is not case insensitive, not sure worth changing it(?) In any case we need to change the case of the request URI (to have them with different case).